include/fiction/io/print_layout.hpp

```diff
@@ -12,7 +12,6 @@
 #include <fmt/color.h>
 #include <fmt/format.h>
 
-#include <algorithm>
```

Comment:
`std::min` and `std::max` are part of `<algorithm>`. Please do not remove this include.

Reply:
I thought they were part of cmath. Thanks for letting me know. I think cmath can be removed then since the calls to floor and ceil were removed earlier.